.github/workflows/daily-release.yml

```diff
@@ -32,21 +32,13 @@ jobs:
           echo "${{ secrets.NPMRC }}" > ~/.npmrc
 
       - name: Download Simple Auth bits
-        uses: nick-invision/retry@v2
-        env:
-          SIMPLE_AUTH_SAS_TOKEN: ${{secrets.SIMPLE_AUTH_SAS_TOKEN}}
-          SIMPLE_AUTH_ENDPOINT: ${{secrets.SIMPLE_AUTH_ENDPOINT}}
-        with:
-          timeout_minutes: 10
-          max_attempts: 5
-          retry_on: error
-          shell: pwsh
-          command: |
-            $version=Get-Content packages/fx-core/templates/plugins/resource/simpleauth/version.txt
-            $fileName="Microsoft.TeamsFx.SimpleAuth_$version.zip"
-            echo $SIMPLE_AUTH_ENDPOINT
-            $url=$env:SIMPLE_AUTH_ENDPOINT+'/'+$fileName+'?'+$env:SIMPLE_AUTH_SAS_TOKEN
-            Invoke-WebRequest $url -OutFile packages/fx-core/templates/plugins/resource/simpleauth/SimpleAuth.zip
+        run: |
```

Comment:
do we need to keep the retry logic in existing workflow? @KennethBWSong do you know the story of existing retry logic?